.github/workflows/docs.yml

```diff
@@ -11,5 +11,5 @@ jobs:
       with:
         docs-folder: "doc/"
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        pre-build-command: "apt install -y graphviz && pip3 install -r requirements.txt && pip3 install . && pip3 install -r qchem/requirements.txt && pip3 install ./qchem"
+        pre-build-command: "apt install -y graphviz && pip3 install -r doc/requirements.txt && pip3 install . && pip3 install -r qchem/requirements.txt && pip3 install ./qchem"
```

Comment:
I wonder how this one snuck in?

Reply:
This was me 😆

Reply:
This actually was just not correct, which I noticed when building locally :D
The docs now don't require the packages in `requirements.txt` but the packages in `doc/requirements.txt` :)